search/tutorials/tutoria_attribute_configuration.md

```diff
@@ -0,0 +1,140 @@
+# **Attribute Configuration Tutorial**
+
+## Let's get started
+
+Retail Search allows you to configure the product attributes for the search purpose. 
+
+If you want the Retail Search to be able to filter and facet your catalog using this product attribute, you should set it as indexable.
+
+If you want to enable the search by some product attribute, you should make this attribute searchable. Note, that onsy text attributes can be searchable.
+
+In this tutorial you will learn how to configure the product attributes to allow the Retail Search to perform products indexing and search.
+
+**Time to complete**: About 4 minutes
+
+## Before you begin
+
+To run Python code samples from this tutorial, you need to set up your virtual environment.
+
+Run the following commands in a terminal:
+```bash
+pip install virtualenv
+```
+```bash
+virtualenv <your-env>
+```
+```bash
+source <your-env>/bin/activate
+```
+Next, install Google packages:
+```bash
+pip install google
+```
+```bash
+pip install google-cloud-retail
+```
+
+**Tip**: Click the copy button on the side of the code box to paste the command in the Cloud Shell terminal and run it.
+
+## Filter by unindexable field
+
+There are several products in the catalog with the attribute "eco-friendly".  The attribute's field **```indexable```** is set to **false**.
```

Comment:
replace "The attribute's field" with "The attribute field"

no need to use possessive pronouns here.